RHDEVDOCS-3918 - Document Alertmanager for user workload monitoring (#46357)
openshift/openshift-docs: modules/monitoring-understanding-alert-routing-for-user-defined-projects.adoc

These notifications are routed by the default Alertmanager instance or, if enabled, an optional Alertmanager instance dedicated to user-defined monitoring.

Users can then create and configure user-defined alert routing by creating or editing the `AlertmanagerConfig` objects for their user-defined projects without the help of an administrator.

After a user has defined alert routing for a user-defined project, user-defined alert notifications are routed as follows:

* To the `alertmanager-main` pods in the `openshift-monitoring` namespace if using the default platform Alertmanager instance.

* To the `alertmanager-user-workload` pods in the `openshift-user-workload-monitoring` namespace if you have enabled a separate instance of Alertmanager for user-defined projects. 

Note the following limitations and features of alert routing for user-defined projects:

* For user-defined alerting rules, user-defined routing is scoped to the namespace in which the resource is defined.
For example, a routing configuration in namespace `ns1` only applies to `PrometheusRules` resources in the same namespace.

> * Cluster administrators continue to define the main Alertmanager configuration by using a custom secret or the {product-title} web console. 

Author: Changed to 

`*  If the user-defined Alertmanager instance is enabled, cluster administrators can also define the main configuration of the user-defined Alertmanager instance.`

= Understanding alert routing for user-defined projects

[role="_abstract"]
As a cluster administrator, you can enable alert routing for user-defined projects. 
With this feature, you can allow users with the **alert-routing-edit** role to configure alert notification routing and receivers for user-defined projects.

> These notifications will then be routed using either the default Alertmanager instance or the optional Alertmanager instance dedicated only to user-defined monitoring, if enabled.

Reviewer: ```suggestion
These notifications are routed by the default Alertmanager instance or, if enabled, by an optional Alertmanager instance dedicated to user-defined monitoring.
```
- Avoids future tense.
- Avoids retrospectively figuring out what "if enabled" applies to.
- Doesn't avoid passive voice.

Author: Edited to read `These notifications are routed by the default Alertmanager instance or, if enabled, an optional Alertmanager instance dedicated to user-defined monitoring.` Much better!

Users can then create and configure user-defined alert routing by creating or editing the `AlertmanagerConfig` objects for their user-defined projects without the help of an administrator.

> 

Author: fixed
